deepreg/model/loss/image.py

```diff
@@ -4,6 +4,7 @@
 import tensorflow as tf
 
 EPS = 1.0e-6  # epsilon to prevent NaN
+PI2_SQRT = 2.506628274631
```

Comment:
it is sqrt(2·pi)

Reply:
@acasamitjana would you like to put a comment to describe it?

or we can do like

```python
import numpy as np
SQRT_2PI = np.sqrt(2*np.pi)
```

this will be probably executed only once.

Reply:
ok, I agree